content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -214,11 +232,32 @@
       outline: dotted 1px;
     }
   }
+
+  .trailheadInner,
+  .trailheadStart {
+    opacity: 0;
+    transition: opacity 0.4s;
```

Comment:
We don't need all the `revealModalContent` related state/methods/setTimeout as we only fade in (and no fade out) and already have a css animation handy:
https://github.com/mozilla/activity-stream/blob/5a8c8f68060ccdf7773502578eaa05559f438264/content-src/styles/_activity-stream.scss#L60-L66

Should be able to just do:
```css
.trailheadInner,
.trailheadStart {
  animation: fadeIn 0.4s;
}
```